graylog2-server/src/main/java/org/graylog2/plugin/inputs/MessageInput.java

```diff
@@ -246,6 +246,10 @@ public Boolean isGlobal() {
         return global;
     }
 
+    public boolean onlyOnePerCluster() {
```

Comment:
Let's add a javadoc comment to this that describes when an input should set this to `true`.